solidity/scripts/circleci-migrate-contracts.sh

```diff
@@ -62,8 +62,12 @@ ssh utilitybox << EOF
   ./node_modules/.bin/truffle migrate --reset --network $TRUFFLE_NETWORK
   echo ">>>>>>FINISH Contract Migration FINISH>>>>>>"
   echo "<<<<<<START Tenderly Push START<<<<<<"
+  # Temporary fix for some odd push issues; OpenZeppelin contracts are
+  # referenced at two paths due to older dependencies, leading to problems
+  # resolving contracts during tenderly push.
```

Comment:
This actually doesn't pull in the `ln -s` for `@openzeppelin`. Are we cool with that?

Reply:
Frankly, I wasn't sure we had the same dependency issue here.

Reply:
Updated to include the symlink